core/src/main/java/org/sonarsource/sonarlint/core/container/connected/update/PluginReferencesDownloader.java

```diff
@@ -80,14 +80,21 @@ public PluginReferences fetchPluginsTo(Path dest, String serverVersion) {
         .setHash(hash)
         .setFilename(filename)
         .build());
-      pluginCache.get(filename, hash, new SonarQubeServerPluginDownloader(key));
     }
     scanner.close();
     PluginReferences pluginReferences = builder.build();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "pluginReferences". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1488)